extensions-contrib/kafka-emitter/src/main/java/org/apache/druid/emitter/kafka/KafkaEmitterConfig.java

```diff
@@ -23,22 +23,49 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.base.Preconditions;
 import com.google.common.collect.ImmutableMap;
+import org.apache.commons.lang3.EnumUtils;
 import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.errors.InvalidConfigurationException;
 
 import javax.annotation.Nullable;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
 public class KafkaEmitterConfig
 {
+  public enum EventType
+  {
+    metrics,
+    alerts,
+    requests,
+    segment_metadata
+  }
+
+  public static final Set<EventType> DEFAULT_EVENT_TYPES = new HashSet<>(Arrays.asList(EventType.alerts, EventType.metrics));
```

Comment:
nit, you can use `ImmutableSet.of(EventType.alerts, EventType.metrics)` to simplify this.